projects/igniteui-angular/src/lib/date-picker/date-picker.component.html

```diff
@@ -1,25 +1,42 @@
 <ng-template #readOnlyDatePickerTemplate>
-    <igx-input-group (click)="openDialog()">
+    <igx-input-group (click)="openDialog()" (mousedown)="mouseDown($event)">
```

Comment:
Not sure if `pointerdown` might be better. Do try on some mobile devices.

Reply:
yes it is working on iPad